src/controllers/reminder.controller.js

```diff
@@ -150,6 +150,30 @@ const reminderController = {
 			})
 		}
 	},
+
+	updateById: async (req, res) => {
+		const { priority, expiryDate, description, shouldRemind } = req.body
+
+		try {
+			const reminderData = { priority, expiryDate, description, shouldRemind }
+
+			const { id } = req.params
+
+			const updateReminder = await db.findByIdAndupdate(id, reminderData)
+
+			if (!id) {
```

Comment:
Why would you put this check here after updating? instead its like this

```javascript
if(!id) {}..
const updateReminder = await db.find...
```

Correct the order

Reply:
oh! that was a mistake, let me quickly correct it